service/storage_proxy.cc

```diff
@@ -2228,10 +2238,13 @@ future<paxos::prepare_summary> paxos_response_handler::prepare_ballot(utils::UUI
                         request_tracker.errors++;
                         paxos::paxos_state::logger.trace("CAS[{}] prepare_ballot: fail to send ballot {} to {}: {}", _id,
                                 ballot, peer, ex);
+                        append_peer_error(request_tracker.errors_message, peer, ex);
                         if (_required_participants + request_tracker.errors > _live_endpoints.size()) {
-                            auto e = std::make_exception_ptr(mutation_write_failure_exception(_schema->ks_name(),
-                                        _schema->cf_name(), _cl_for_paxos, summary.committed_ballots_by_replica.size(),
-                                        request_tracker.errors, _required_participants, db::write_type::CAS));
+                            auto e = std::make_exception_ptr(mutation_write_failure_exception(
+                                format("Failed to prepare ballot {} for {}.{}. Replica errors: {}",
+                                    ballot, request_tracker.errors_message, _schema->ks_name(), _schema->cf_name()),
```

Comment:
It looks like `request_tracker.errors_message` should be passed last

Reply:
fixed